index.md

```diff
@@ -99,6 +114,22 @@ Read the full [Berlin Code of Conduct here](https://berlincodeofconduct.org/).
 
 </div>
 
+<div class="content-section content-section--whitebg sponsors" markdown="1">
+
+## Sponsors
+
+<div class="sponsors__listing sponsors__listing--gold" markdown="1">
+[![sum.cumo](assets/images/sponsors/sumcumo.svg)](https://www.sumcumo.com/)
```

Comment:
why listing sum.cumo twice?

Reply:
Because the first one is for gold only and the last section are all current sponsors

Reply:
@janz93 I am with @emrox here. One mention per Sponsor on the main page is enough from my point of view. Same procedure as last year. :)

Reply:
Okay I guess now we have enough sponsors to seperate them again.
I'll update the PR

Reply:
@skrings and @emrox I updated the sponsor section and divided them as we did it last year.
@skrings if you have any better wording for the community sponsor let me know.
My goal is the have this merged by Wednesday before the ruby on ice on Munich is started